tests/implementor/conftest.py

```diff
@@ -164,6 +187,19 @@ def platform_blocked_fault_configuration(
     )
 
 
+@pytest.fixture
+def another_platform_blocked_fault_configuration(
+    platform: Platform,
+) -> PlatformBlockedFaultConfiguration:
+    return PlatformBlockedFaultConfiguration.create(
+        start_tick=20,
+        end_tick=200,
+        description="test PlatformBlockedFault",
+        affected_element_id=platform.identifier,
+        strategy="regular",
+    )
```

Comment:
Couldn't you use `platform_blocked_fault_configuration_data`?

Reply:
Good idea! I'll change that!